### ISSUE TITLE
请问支持swift 4.2了吗？

### ISSUE THREAD

**Issue by meifu1709** [jiecao-fm/SwiftTheme#80]

**Comment by FormioCour** [comment]
可暂时用这个  
```
pod 'SwiftTheme', :git => 'https://github.com/jiecao-fm/SwiftTheme.git', :commit => 'fbfb3bf80add22db628cda01520ca08214d99486'
```

**Comment by kcramer** [comment]
Swift 4.2 support is included in 0.4.3.  I'm closing this issue.  If you have any issues, feel free to open a new issue.
